docs/modules/sql/pages/mapping-to-jdbc.adoc

```diff
@@ -76,4 +86,100 @@ OPTIONS (
 ```
 
 <1> The name of the connector.
-<2> The name of the external data store configuration on your members.
\ No newline at end of file
+<2> The name of the external data store configuration on your members.
+
+== Data Type Mapping Between Hazelcast and MSSQL
+Hazelcast supports a subset of SQL data types. The following MSSQL types are mapped to the respective Hazelcast SQL type:
```

Comment:
@Edizc Do you have some source for the list of MS SQL data types? It would help many people (internal & external) if we have a link to official MS SQL docs here. (Same for Oracle below).